07-modeling.Rmd

```diff
@@ -121,9 +125,9 @@ des_obj %>%
   )
 ```
 
-where `des_obj` is a design object, `outcomevar` is the outcome variable, `groupvar` is the group variable, and `na.action=na.omit` is set so that records with missing data in the outcome or group variable are removed for prediction ^[See `help(na.omit)` for more information on options to use for `na.action`]. The function `svyglm()` does not have the design as the first argument so the `.` notation is used to pass it with a pipe.
+where `des_obj` is a design object, `outcomevar` is the outcome variable, `groupvar` is the group variable, and `na.action=na.omit` is set so that records with missing data in the outcome or group variable are removed for prediction^[See `help(na.omit)` for more information on options to use for `na.action`]. The function `svyglm()` does not have the design as the first argument so the dot (`.`) notation is used to pass it with a pipe (see Chapter \@ref(c06-statistical-testing) for more details).
```

Comment:
Should we reference Chapter 4 in the footnote since there is text in there about handling missing data.